.gitignore

```diff
@@ -1,3 +1,4 @@
 build
 node_modules
 npm-debug.log
+.idea/
```

Comment:
(Shameless plug for a tool I built): Using WebStorm? Care to develop a plugin for import-js, a tool I've built to help importing/requiring javascript dependencies? There are plugins for Emacs, vim, and Sublime. But none for WebStorm (yet). https://github.com/trotzig/import-js/issues/102